src/App.scss

```diff
@@ -12,14 +17,42 @@ code {
     monospace;
 }
 
-.App {
+.App, .BusStop {
```

Comment:
Could be good to separate out scss files for each component. That's the industry standard

Reply:
i do! busstop is imported in busstop component